moped-editor/src/components/LookupSelectComponent.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { MenuItem, Select } from "@material-ui/core";
+import { handleKeyEvent } from "../utils/materialTableHelpers";
+
+/**
+ * Component for dropdown select using a lookup table as options
+ * @param {*} props
+ * @returns {React component}
+ */
+const LookupSelectComponent = (props) => (
+  <Select
+    style={{ minWidth: "8em" }}
+    id={props.name}
+    value={props.value || props.defaultValue}
+  >
+    {props.data.map((item) => (
+      <MenuItem
+        onChange={() => props.onChange(item[`${props.name}_id`])}
+        onClick={() => props.onChange(item[`${props.name}_id`])}
+        onKeyDown={(e) => handleKeyEvent(e)}
+        value={item[`${props.name}_id`]}
+        key={item[`${props.name}_name`]}
+      >
+        {item[`${props.name}_name`]}
+      </MenuItem>
+    ))}
+    {props.columnDef.title === "Program" && (
+      <MenuItem
+        onChange={() => props.onChange("")}
+        onClick={() => props.onChange("")}
+        onKeyDown={(e) => handleKeyEvent(e)}
+        value=""
+      >
+        -
```

Comment:
I noticed that I don't see this "-" option come up for the Program column like it does in staging. Is that a new change?

Reply:
I didn't include it because its my understanding that the reason we had it in the select dropdown, was if someone wanted to edit an existing record and remove the program, the only way to do it was to select "-". But with autocomplete, you can use the X to clear the field. unless there was another reason to include "-", then I am happy to put it back in

Reply:
Cool - that makes total sense to me - just curious! 🙏